dev/io.openliberty.restfulWS.3.0.cdi.3.0_fat/test-applications/lifecyclemismatch/src/io/openliberty/restfulWS30/cdi30/fat/lifecyclemismatch/ApplicationScopedResource.java

```diff
@@ -0,0 +1,141 @@
+/*******************************************************************************
+ * Copyright (c) 2021 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.restfulWS30.cdi30.fat.lifecyclemismatch;
+
+import java.io.Serializable;
+
+import io.openliberty.restfulWS30.cdi30.fat.lifecyclemismatch.simpleresource.Person;
+import io.openliberty.restfulWS30.cdi30.fat.lifecyclemismatch.simpleresource.SimpleBean;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+import jakarta.ws.rs.Consumes;
+import jakarta.ws.rs.DELETE;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.POST;
+import jakarta.ws.rs.PUT;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+
+/**
+ * <code>HelloWorldResource</code> is a simple POJO which is annotated with
+ * JAX-RS annotations to turn it into a JAX-RS resource.
+ * <p/>
```

Comment:
```suggestion
 * <p>
```

many javadoc tools will complain about this - even though it is proper XHTML syntax.